app/src/main/java/com/example/easycloset/HomeFragment.java

```diff
@@ -55,5 +78,58 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         tvMaxTemp = view.findViewById(R.id.tvMaxTemp);
         tvSunrise = view.findViewById(R.id.tvSunrises);
         tvSunset = view.findViewById(R.id.tvSunsets);
+        nestedScrollView = view.findViewById(R.id.homeScreen);
+    }
+
+    public void setWeather(Weather weather) {
+        this.weather = weather;
+    }
+
+    public void setLatitude(double latitude) {
+        this.latitude = latitude;
+    }
+
+    public void setLongitude(double longitude) {
+        this.longitude = longitude;
+    }
+
+    public double getLatitude() {
+        return latitude;
+    }
+
+    public double getLongitude() {
+        return longitude;
+    }
+
+    public void populateHomeFragment() {
+        AsyncHttpClient client = new AsyncHttpClient();
+        String apiByLat = "https://api.openweathermap.org/data/2.5/weather?lat=" + getLatitude() + "&lon=" + getLongitude() + "&units=imperial&appid=942dd77fa358eb3439a8212cb16724cd";
+
+        client.get(apiByLat, new JsonHttpResponseHandler() {
```

Comment:
you start call here, but you never cancel the call if fragment get stopped, detached, destroyed etc. You might want to do it, otherwise you onSuccess/onFailure might be called in background. If this would happen then for example progressDialog.dismiss(); might produce a crash (I'm not 100% sure about that though)